apps/aehttp/priv/swagger.json

```diff
@@ -741,7 +765,19 @@
         "description" : "Get the genesis block transactions count",
         "operationId" : "GetGenesisBlockTxsCount",
         "produces" : [ "application/json" ],
-        "parameters" : [ ],
+        "parameters" : [ {
+          "name" : "tx_types",
+          "in" : "query",
+          "description" : "Transactions types to show. Comma separated",
+          "required" : false,
+          "type" : "string"
```

Comment:
would an enum not feel much more natural? Since it is such a rich type, we could put it in the body instead of parameters.... I am not an interface expert, but putting everything in parameters is not necessarily needed.

Reply:
I'd love to have a specific type for that param. It is just that swagger does not support having complex types in the query params.
OTOH I don't like the idea of splitting params between the query params and the body, esp in a GET request.